pkg/cmd/experimental/registry/registry.go

```diff
@@ -140,6 +143,18 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 		label = valid
 	}
 
+	nodeSelector := map[string]string{}
+	if len(cfg.Selector) > 0 {
+		valid, remove, err := app.LabelsFromSpec(strings.Split(cfg.Selector, ","))
+		if err != nil {
+			glog.Fatal(err)
```

Comment:
I copied this from the router. Looks like there is a discrepancy between the 2. Router calls `glog.Fatal` whereas Registry returns err.